api/.eslintrc.json

```diff
@@ -0,0 +1,80 @@
+{
+    "env": {
+        "node": true
+    },
+    "parser": "@babel/eslint-parser",
```

Comment:
made a copy of the original .eslintrc.json because the original wants a react plugin which node.js files do not need